src/EditorFeatures/Test2/FindReferences/MockStreamingFindReferencesPresenter.vb

```diff
@@ -0,0 +1,30 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports System.Composition
+Imports Microsoft.CodeAnalysis.Editor.Host
+Imports Microsoft.CodeAnalysis.FindUsages
+Imports Microsoft.CodeAnalysis.Host.Mef
+
+Namespace Microsoft.CodeAnalysis.Editor.UnitTests.FindReferences
+    <Export(GetType(IStreamingFindUsagesPresenter))>
+    <[Shared]>
+    <PartNotDiscoverable>
+    Friend Class MockStreamingFindReferencesPresenter
```

Comment:
So we have two IStreamingFindUsagesPresenter now? Can we unify these, or at least name them in some way that explains the distinction?

Reply:
This change is now removed from this pull request.

Reply:
*This* change is removed, but a similar feature is provided in a different way now.